packages/webcomponents/src/components/payout-transactions-list/payout-transactions-table.tsx

```diff
@@ -0,0 +1,155 @@
+  ),
+  description: () => (
+    <th part={tableHeadCell} scope="col" title="The description of each transaction">
+      Description
+    </th>
+  ),
+  source_id: () => (
+    <th part={tableHeadCell} scope="col" title="The unique identifier of the source object associated with the payment balance transaction">
+      Source ID
+    </th>
+  ),
+  txn_type: () => (
+    <th part={tableHeadCell} scope="col" title="The type of each transaction">
+      Type
+    </th>
+  ),
+  created_at: () => (
+    <th part={tableHeadCell} scope="col" title="The date and time each transaction was made">
+      Processed On
+    </th>
+  ),
+  available_on: () => (
+    <th part={tableHeadCell} scope="col" title="The date and time each transaction will be available">
+      Available On
+    </th>
+  ),
+  updated_at: () => (
+    <th part={tableHeadCell} scope="col" title="The date and time each transaction was last updated">
+      Updated On
+    </th>
+  ),
+};
+
+export const payoutTransactionTableCells = {
+  id: (transaction: PayoutBalanceTransaction) => (
+    <td part={getAlternateTableCellPart(transaction.id)}>
+      {transaction.id}
+    </td>
+  ),
+  account_id: (transaction: PayoutBalanceTransaction) => (
+    <td part={getAlternateTableCellPart(transaction.account_id)}>
+      {transaction.account_id}
+    </td>
+  ),
+  payout_id: (transaction: PayoutBalanceTransaction) => (
+    <td part={getAlternateTableCellPart(transaction.payout_id)}>
+      {transaction.payout_id}
+    </td>
+  ),
+  financial_transaction_id: (transaction: PayoutBalanceTransaction) => (
+    <td part={getAlternateTableCellPart(transaction.financial_transaction_id)}>
+      {transaction.financial_transaction_id}
+    </td>
+  ),
+  amount: (transaction: PayoutBalanceTransaction) => (
+    <td part={getAlternateTableCellPart(transaction.amount)}>
+      {transaction.formattedPaymentAmount(transaction.amount)}
+    </td>
+  ),
+  fee: (transaction: PayoutBalanceTransaction) => (
+    <td part={getAlternateTableCellPart(transaction.fee)}>
+      {transaction.formattedPaymentAmount(transaction.fee)}
+    </td>
+  ),
+  net: (transaction: PayoutBalanceTransaction) => (
+    <td part={getAlternateTableCellPart(transaction.net)}>
+      {transaction.formattedPaymentAmount(transaction.net)}
+    </td>
+  ),
+  currency: (transaction: PayoutBalanceTransaction) => (
+    <td part={getAlternateTableCellPart(transaction.currency)}>
+      {transaction.currency}
+    </td>
+  ),
+  description: (transaction: PayoutBalanceTransaction) => (
+    <td part={getAlternateTableCellPart(transaction.description)}>
+      {transaction.description}
+    </td>
+  ),
+  source_id: (transaction: PayoutBalanceTransaction) => (
+    <td part={getAlternateTableCellPart(transaction.source_id)}>
+      {transaction.source_id}
+    </td>
+  ),
+  txn_type: (transaction: PayoutBalanceTransaction) => (
+    <td part={getAlternateTableCellPart(transaction.txn_type)}>
+      {transaction.txn_type}
```

Comment:
@colganmeanor Can we translate these using the type -> name mapping I created for payments transactions list?